source/_patterns/00-atoms/components/info-bar.mustache

```diff
@@ -1,3 +1,14 @@
-<div class="info-bar info-bar--{{type}}">
-  <span class="info-bar__text">{{{text}}}</span>
+<div class="info-bar info-bar--{{type}}" id="{{id}}"
+     {{#dismiss}}
+        data-behaviour="InfoBar"
+        {{#cookieDuration}} data-cookie-name-root="infoBar_" data-cookie-duration="{{cookieDuration}}"{{/cookieDuration}}
+        {{#cookieExpires}} data-behaviour="InfoBar" data-cookie-name-root="infoBar_" data-cookie-expires="{{cookieExpires}}"{{/cookieExpires}}>
```

Comment:
`data-behaviour="InfoBar"` duplicated?

Reply:
Removed in https://github.com/elifesciences/pattern-library/pull/958/commits/8a19524ed9f55735cb986edac6152ddb27800b21